app/src/main/res/xml/logins_preferences.xml

```diff
@@ -12,9 +12,9 @@
         android:defaultValue="true"
         android:key="@string/pref_key_autofill_logins"
         android:title="@string/preferences_passwords_autofill" />
-    <Preference
-        android:key="@string/pref_key_password_sync_logins"
-        android:summary="@string/preferences_passwords_sync_logins_off"
+    <org.mozilla.fenix.settings.SyncPreference
+        android:defaultValue="true"
```

Comment:
Does this do anything? Just wondering if this is needed.

Reply:
good point. removed